documentation/source/rescap.rst

```diff
@@ -0,0 +1,120 @@
+###############################
+The cocotb ``rescap`` Testbench
+###############################
+
+.. versionadded:: 1.5
+
+This is the testbench :mod:`test_rescap` for the design ``rescap`` showing
+how cocotb can be used in an analog-mixed signal (AMS) simulation.
+
+******************
+Overview Schematic
+******************
+
+.. image:: rescap_schematic.png
+
+
+**********
+The Design
+**********
+
+The design consists of a resistor and capacitor model (both written in Verilog-AMS) connected
+in series in a SystemVerilog module, as shown here:
+
+.. code-block:: none
+
+          i_resistor             i_capacitor
+  vdd >---o--v^v^v--o------------o---||---o---< vss
+          p         n    vout    p        n
+
+
+The code of the design is:
```

Comment:
This is a little awkard. The last sentence of the previous paragraph works if you just remove the ASCII art and this paragraph.

Reply:
My last name rhymes with "awkward" :)